fillBackground.py

```diff
@@ -0,0 +1,112 @@
+import cv2
+import numpy as np 
+
+from tqdm import tqdm
+from matcher import matcher
+
+class fillBackGround:
```

Comment:
For class, we always use Pascal case. Change class name to BackgroundFilling
Also, file name should be snake case, ex: background_filling.py

Reply:
done, Sir